python/paddle/v2/fluid/tests/test_reorder_lod_tensor.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+import paddle.v2.fluid as fluid
+import numpy
+
+
+class TestReorderLoDTensor(unittest.TestCase):
+    def test_reorder(self):
```

Comment:
Please add Gradient test

Reply:
It has been tested, since IG.sum() == 1 and LoD can be restored.